reports/performance/_outliers.qmd

```diff
@@ -392,7 +428,7 @@ qnt_df <- assessment_pin %>%
     `Est. FMV` = pred_pin_final_fmv_round
   ) %>%
   mutate(
```

Comment:
These should align with changes seen throughout, with an improvement that Billy already made.